multiworld/envs/mujoco/sawyer_reach_torque/sawyer_reach_torque_env.py

```diff
@@ -265,7 +254,7 @@ def compute_rewards(self, actions, obs):
         return r
 
     def set_to_goal(self, goal):
-        raise NotImplementedError()
+        pass
```

Comment:
Please undo this. It's good to get in a "defensive programming" mindset, where you optimize for preventing bugs (raising exception, assert statements, readable code, etc.)

If anything, we could just remove this function.